frontend/src/app/Settings/Users/ReactivateUserModal.tsx

```diff
@@ -14,11 +17,20 @@ interface Props {
   user: SettingsUser;
 }
 
+export const ORG_ADMIN_REACTIVATE_COPY =
+  "Are you sure you want to reactivate this account?";
+export const SITE_ADMIN_REACTIVATE_COPY =
+  "When you reactivate their account, the user will need to choose a new password. Do you want to reactivate this account?";
+
 const ReactivateUserModal: React.FC<Props> = ({
   onClose,
   onReactivateUser,
   user,
 }) => {
+  const loggedInUser = useSelector<RootState, User>((state) => state.user);
+  const loggedInUserIsSiteAdmin =
+    loggedInUser.roleDescription.includes(SITE_ADMIN_ROLE_DESC);
```

Comment:
I wonder if this is going to work from the support admin manage users page
I think if you're not ghosted as an admin the role description might be "Super Admin"
this is based on my skimming code, esp looking at [this method](https://github.com/CDCgov/prime-simplereport/blob/f1d6ba6453023bc09d5d75675efd09939671dcaa/backend/src/main/java/gov/cdc/usds/simplereport/api/model/User.java#L38)
I'll try to reproduce in some way

Reply:
if I log in to prod `whoami` returns "Super Admin" as the role description for me. could we use the `isAdmin` flag here?

Reply:
Ahh thank you for flagging that! Don't know how I missed it.